internal/servicelog/template.go

```diff
@@ -54,6 +55,10 @@ func (m *Message) GetSubscriptionID() string {
 	return m.SubscriptionID
 }
 
+func (m *Message) GetDocReferences() []string {
```

Comment:
Getters aren't really needed for exported types, but I wanted to maintain consistency with the rest of the file.